packages/TelegramClient-UI.package/TCUNotification.class/instance/messageText.st

```diff
@@ -0,0 +1,6 @@
+drawing
+messageText
+
+	^ TCUEmojiHelper embedEmojisInto:
+			((self message asText withNoLineLongerThan: self class defaultLineLength) truncateWithElipsisTo:
+				self class defaultMessageLength)
```

Comment:
I would put `truncateWithEllipsisTo:` on the next line as well

Reply:
Also used the same styling in `TCUChatListItem->lastMessageText`.